hokusai/development.yml

```diff
@@ -2,16 +2,18 @@ version: "2"
 services:
   metaphysics:
 {% include 'templates/docker-compose-service.yml.j2' %}
-    environment:
-      - MEMCACHED_URL=metaphysics-memcached:11211
+    command: ["yarn", "start"]
     env_file:
       - ../.env.shared
+      - ../.env.development
       - ../.env
-    ports:
-      - 3000:3000
     depends_on:
       - metaphysics-memcached
+    ports:
+      - 5001:5001
+    volumes:
+      - ../:/app
   metaphysics-memcached:
-    image: memcached
+    image: memcached:1.4.34-alpine
     ports:
-      - 11211:11211
+      - 11212:11211
```

Comment:
This mapping conflicts with gravity memcached service so I incremented the last port number. Before doing this, the `metaphysics-memcached` container would not successfully run and just hang while starting.

Reply:
See above comment on `env.development`.

Reply:
Removed this change in https://github.com/artsy/metaphysics/pull/5341/commits/81875a59092313885eed3b7ff52540f9ccd962f6